File: example/sample_app.py
def rectangle(w, h):
    return w * h

def square(s):
    return rectangle(s, s)

def triangle(b, h):
    return rectangle(b, h) * .5

def mecard(name, email, phone):
    return f'MECARD:N:{name};EMAIL:{email};TEL:{phone};'

print(rectangle(5, 3))
print(square(5))
print(triangle(5, 3))
print(mecard('Peter', 'peter@marvel.com', '088-123-7788'))

File: example/sample_app.js
'use strict';

function rectangle(w, h) {
    return w * h
}

function square(s) {
    return rectangle(s, s)
}

function triangle(b, h) {
    return rectangle(b, h) * .5
}

function mecard(name, email, phone) {
    return `MECARD:N:${name};EMAIL:${email};TEL:${phone};`
}

console.log(rectangle(5, 3));
console.log(square(5));
console.log(triangle(5, 3));
console.log(mecard('Peter', 'peter@marvel.com', '088-123-7788'))
